src/mail/view/MailViewer.js

```diff
@@ -1792,6 +1790,19 @@ export class MailViewer {
 	}
 }
 
+/**
+ * support and invoice mails can contain links to the settings page.
+ * in the desktop app, all normal anchor links are opened with the default browser.
+ * this fails for relative links because the desktop app prepends file://C:/ or file:///
+ * and we prevent any files being opened from links in mails.
+ * */
+function isSettingsLink({href, origin}: HTMLAnchorElement, mail: Mail): boolean {
+	console.log(href, origin)
+	//Electron in windows appends /<drive letter>:/ to relative urls, so we should match with or without
+	const isSettingsHref = href.startsWith(origin) && !!href.slice(origin.length).match(/^(\/[A-Z]:)?\/settings\//)
```

Comment:
While I think the right way would be to have a special attribute on a link so that we would easily intercept it (because we control email template anyway) we can still work with what we have now.

While HTMLLinkElement.prototype.href returns interpreted abosolute path, `linkElement.getAttribute("href")` returns raw HTML value so we can easily match it against `'/settings/invoice'`.

Reply:
adding an attribute to the link in the template seems easy to forget if you're writing support responses and want to link someplace in the settings. The combination of team mail + the exact return value of getAttribute starting with `/settings/` should be the most versatile and robust solution I think

Reply:
I don't think we ever write links by hand, we avoid sending any links generally. This is rather for invoice emails and is hardcoded in the place where email is assembled. I think checking for href is okay too though.